Sources/swift-doc/Subcommands/Diagram.swift

```diff
@@ -11,23 +11,27 @@ extension SwiftDoc {
         struct Options: ParsableArguments {
             @Argument(help: "One or more paths to Swift files")
             var inputs: [String]
+
+            @Option(name: .shortAndLong,
```

Comment:
I think that we should support only the long form of this option flag.

```suggestion
            @Option(name: .long,
```